docs/ref/apis/vpc.md

```diff
@@ -1134,5 +1138,6 @@ _Appears in:_
 | `defaultSNATIP` _string_ | Default SNAT IP for Private Subnets. |  |  |
 | `loadBalancerIPAddresses` _string_ | LoadBalancerIPAddresses (AVI SE Subnet CIDR or NSX LB SNAT IPs). |  |  |
 | `privateIPs` _string array_ | Private CIDRs used for the VPC. |  |  |
+| `supportedNetworkingStack` _string_ | supportedNetworkingStack indicates the networking stack supported by the VPC.<br />Valid values: FULL_STACK_VPC, VLAN_BACKED_VPC |  |  |
```

Comment:
I heard NSX decides to use "NetworkStack", not "SupportedNetworkingStack". Could you check?